bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/UpdateCookieCmdTest.java

```diff
@@ -192,9 +192,12 @@ private void updateCookie(String option, String optionVal, boolean useHostNameAs
         Assert.assertEquals("Wrongly updated cookie!", useHostNameAsBookieID, !cookie.isBookieHostCreatedFromIp());
         verifyCookieInZooKeeper(newconf, 1);
 
-        File journalDir = Bookie.getCurrentDirectory(conf.getJournalDir());
-        Cookie jCookie = Cookie.readFromDirectory(journalDir);
-        jCookie.verify(cookie);
+        for (File journalDir : conf.getJournalDirs()) {
+            journalDir = Bookie.getCurrentDirectory(journalDir);
+            Cookie jCookie = Cookie.readFromDirectory(journalDir);
+            jCookie.verify(cookie);
+        }
+
         File[] ledgerDir = Bookie.getCurrentDirectories(conf.getLedgerDirs());
```

Comment:
Looks like tests for this feature are missing.
- At lest few of the journal tests needs to be implemented with multiple journals 
- Looks like ; if there are multiple journal stanzas with NULL, that many journal files are created in the current dir; not sure if that is intended; also no checks for uniqueness of each journal dir